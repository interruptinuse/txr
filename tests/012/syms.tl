(load "../common")

(defpackage fb-2
  (:local loc-0 loc-1 loc-2)
  (:fallback usr))

(defpackage fb-1
  (:local loc-0 loc-1)
  (:fallback fb-2 usr))

(defpackage main
  (:local loc-0)
  (:fallback fb-1 fb-2 usr))

(in-package fb-2)

(prinl '(loc-0 loc-1 loc-2 cons))

(in-package fb-1)

(prinl '(loc-0 loc-1 loc-2 cons))
(prinl '(fb-2:loc-0 fb-2:loc-1 fb-2:loc-2 fb-2:cons))

(in-package main)

(prinl '(loc-0 loc-1 loc-2 cons))
(prinl '(fb-2:loc-0 fb-2:loc-1 fb-2:loc-2 fb-2:cons))
(prinl '(fb-1:loc-0 fb-1:loc-1 fb-1:loc-2 fb-1:cons))
