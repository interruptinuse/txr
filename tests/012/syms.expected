(loc-0 loc-1 loc-2 cons)
(loc-0 loc-1 loc-2 cons)
(fb-2:loc-0 fb-2:loc-1 loc-2 cons)
(loc-0 loc-1 loc-2 cons)
(fb-2:loc-0 fb-2:loc-1 loc-2 cons)
(fb-1:loc-0 loc-1 loc-2 cons)
